### ISSUE TITLE
Merge timeseries on dashboard frontend

### ISSUE THREAD

**Issue by benshayden** [catapult-project/catapult#3818]
Currently, the story-tag-filter only filters the story picker autocomplete box, it doesn't affect the charts.
Telemetry merges values across all stories in a benchmark before they are uploaded to the dashboard, so users can see summarized metrics across all stories in a benchmark.
There is no way for users to see summarized metrics across *some* of the stories in a benchmark.
As benchmarks are being merged, it makes less sense to summarize metrics across all stories in a benchmark, but summarizing across all stories with a selected tag (or a few tags) would be useful.
It may also be useful to summarize metrics across a few selected bots.
Computing **all** possible combinations of story tags and bots, and merging all timeseries across each combination, is infeasible anywhere in the dashboard backend.
Merging timeseries on demand in the frontend is feasible.
The Histogram Pipeline is adding more statistics to Row objects in the dashboard database.
GraphJsonHandler can return timeseries containing all of these statistics.
On the frontend, these statistics can be reconstructed into [RunningStatistics](https://github.com/catapult-project/catapult/blob/master/tracing/tracing/base/math/running_statistics.html) objects, which is the same object used by Histograms when computing metrics and merging Histograms.
RunningStatistics objects can be merged by zipping across timeseries.
To recap, users can select story tags or other subsets of stories or multiple bots, charts can request /graph_json to retrieve all selected timeseries, reconstruct each point in each timeseries into RunningStatistics objects, merge RunningStatistics dynamically across timeseries to produce summarized metrics in the frontend, without any combinatoric explosion in the backend.
Alternatively, GraphJsonHandler could reconstruct and merge the RunningStatistics before returning the summarized timeseries to the frontend, but this would unnecessarily risk hitting a request timeout, and miss the opportunity to cache the individual timeseries in the frontend in case users want to drill down from summarized timeseries to individual timeseries, say for a single story or bot.
